### DISCUSSION TITLE
QR settings for only video mode? (and not photo mode) - Hero11 Black

### DISCUSSION THREAD

**Discussion by Fedess1986** [gopro/labs#966] [Q&A]
I don't use gopro much for photos but realized that taking few photos with LOGB400 + WIDE enabled (which I use a lot for videos) also applies for photo mode. Is there a way to keep the settings only for video mode?

**Comment by dnewman-gpsw** [comment]
No, all the four CC commands like LOGB=x are global, as they operate (hack) the camera feature as such a low level, it is not easy to separate camera modes.

**Reply by Fedess1986** [reply]
Thanks David!
